Wirtualnik.Frontend/components/common/PopupMenu.vue

```diff
@@ -96,58 +100,74 @@ export default class PopupMenu extends Vue {
         return event;
     }
 
+    logout() {
+        this.$store.commit(`auth/${TokenMutations.SET_TOKEN}`, '');
+    }
 
-    logout(){
-      this.$store.commit(`auth/${TokenMutations.SET_TOKEN}`, '')
-
+    login() {
+        var token = '';
+        var that = this;
```

Comment:
Functions declared using arrow functions does not change the scope of self-reference, therefore `this` keyword can be used after the function declaration is updated (see below)